resume-buddy/src/main/java/com/google/sps/api/Email.java

```diff
@@ -5,25 +5,38 @@
 import com.google.api.services.gmail.model.Message;
 import com.google.sps.Consts;
 import java.io.ByteArrayOutputStream;
+import java.io.File;
 import java.io.IOException;
 import java.util.Properties;
+import javax.activation.DataHandler;
+import javax.activation.DataSource;
+import javax.activation.FileDataSource;
 import javax.mail.MessagingException;
+import javax.mail.Multipart;
 import javax.mail.Session;
 import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeBodyPart;
 import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
 
 /** Class responsible to create an email and send it to recipient */
 public class Email {
```

Comment:
Challenge: Write tests for the functions defined in this class. I would file a GitHub issue for this and add a TODO to add tests in a later PR.